dlt/common/storages/load_package.py

```diff
@@ -723,19 +723,12 @@ def build_job_file_name(
 
     @staticmethod
     def is_package_partially_loaded(package_info: LoadPackageInfo) -> bool:
```

Comment:
the behavior is unified now between the different package states, I'd say this is correct.